src/Abp.Web/Web/Constants/IConstantScriptManager.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Abp.Web.Constants
+{
+    /// <summary>
+    /// Define interface to get constants and enumerations injected into a script
+    /// </summary>
+    public interface IConstantScriptManager
+    {
+        /// <summary>
+        /// Gets Javascript that contains setting values.
+        /// </summary>
+        string GetScript();
```

Comment:
This should possibly also be async to follow the other script manager interfaces' convention (see ISettingScriptManager for example)